app-editors/levee/levee-4.0.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="Really tiny vi clone, for things like rescue disks"
+HOMEPAGE="https://www.pell.portland.or.us/~orc/Code/levee/"
+SRC_URI="https://www.pell.portland.or.us/~orc/Code/levee/${P}.tar.bz2"
+
+LICENSE="levee"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+
+RDEPEND="
+	!app-text/lv
+	sys-libs/ncurses:0=
+"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-fix-build-for-clang16.patch
+	"${FILESDIR}"/${P}-tgetent-ncurses-gentoo.patch
+)
+
+src_configure() {
+	export AC_CPP_PROG="$(tc-getCPP)"
+	export AC_PATH="${PATH}"
+	export AC_LIBDIR="$($(tc-getPKG_CONFIG) --libs curses ncurses termcap)"
```

Comment:
Maybe this should be --libs-only-L?

Reply:
Because AC_LIBDIR should rather be a path than something like -lncurses and such, right? 
Well I think so, too. This is from the old ebuild. I do not know why it was done like this. I fact the ebuild works without that whole line, because it falls back to /usr/lib I think 
```
AC_PREFIX=${AC_PREFIX:-/usr/local}
    AC_EXECDIR=${AC_EXECDIR:-$AC_PREFIX/bin}
    AC_SBINDIR=${AC_SBINDIR:-$AC_PREFIX/sbin}
    AC_LIBDIR=${AC_LIBDIR:-$AC_PREFIX/lib}
    AC_MANDIR=${AC_MANDIR:-$AC_PREFIX$__share/man}
    AC_LIBEXEC=${AC_LIBEXEC:-$AC_PREFIX/$__libexec}
    AC_CONFDIR=${AC_CONFDIR:-/etc}
```

Reply:
Exactly -- but we don't want to assume /usr/lib, as it'll break LLD.